code/modules/planar_lighting/light_mob.dm

```diff
@@ -1,28 +1,28 @@
 /mob
-	var/image/master_plane/master_plane
-	var/image/dark_plane/dark_plane
+	var/obj/screen/master_plane/master_plane
+	var/obj/screen/dark_plane/dark_plane
 
 /mob/Login()
 	..()
-	master_plane = new(loc=src)
-	dark_plane = new(loc=src)
+	master_plane = new()
+	dark_plane = new()
 	if(client)
-		client.images += master_plane
-		client.images += dark_plane
+		client.screen += master_plane
+		client.screen += dark_plane
 	update_env_light()
 
 /mob/dead/observer/Login()
 	..()
 	if (!seedarkness)
-		client.images -= master_plane
-		client.images -= dark_plane
+		client.screen -= master_plane
+		client.screen -= dark_plane
 
 /mob/dead/observer/updateghostsight()
 	..()
 	if(client)
 		if(!seedarkness)
```

Comment:
Could move this to a proc in the plane object itself which just sets `opacity` to 0, effectively turning the lighting invisible without even touching any lists.